File: android/src/main/java/com/getcapacitor/community/classes/options/StartOptions.java
package com.getcapacitor.community.classes.options;

import androidx.annotation.Nullable;
import com.getcapacitor.PluginCall;

public class StartOptions {

    @Nullable
    private String message;

    public StartOptions(PluginCall call) {
        @Nullable
        String message = call.getString("message");
        this.setMessage(message);
    }

    private void setMessage(@Nullable String message) {
        this.message = message;
    }

    @Nullable
    public String getMessage() {
        return message;
    }
}

File: ios/Sources/BitchatPlugin/Classes/Options/StartOptions.swift
import Foundation
import Capacitor

@objc public class StartOptions: NSObject {
    private var message: String?

    init(_ call: CAPPluginCall) {
        super.init()

        if let message = call.getString("message") {
            self.setMessage(message)
        }
    }

    func setMessage(_ message: String?) {
        self.message = message
    }

    func getMessage() -> String? {
        return self.message
    }
}
